IFC4x3/PropertyConstants/c/CONVEX_CONVEX_2aOB10Pv993OdO4W9RnUic.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="CONVEX_CONVEX_2aOB10Pv993OdO4W9RnUic" Name="CONVEX_CONVEX" UniqueId="a460b040-6792-490d-89d8-12025bc5eb26" />
```

Comment:
Is this one properly named? Should it not be just CONVEX? Description.

Reply:
This is from the domain.